float64 isOut
string stepmode
float64 x
float64 y
float64 z
float64 a
float64 b
float64 c
float64 j1
float64 j2
float64 j3
float64 j4
float64 j5
float64 j6
bool isSend
bool isFinish
